src/GitVersion.Core/Helpers/ValueFormatter.cs

```diff
@@ -0,0 +1,36 @@
+namespace GitVersion.Helpers;
+
+internal static class ValueFormatter
+{
+    private static readonly List<IValueFormatter> _formatters =
```

Comment:
Private fields aren't prefixed in the GitVersion codebase.

```suggestion
    private static readonly List<IValueFormatter> formatters =
```

Reply:
Amen, you make me happy. Done, editorconfig didn't highlight.